ethereum/contracts/BeefyLightClient.sol

```diff
@@ -442,4 +347,157 @@ contract BeefyLightClient {
                 THRESHOLD_DENOMINATOR -
                 1) / THRESHOLD_DENOMINATOR;
     }
+
+    function verifyCommitment(
+        uint256 id,
+        Commitment calldata commitment,
+        ValidatorProof calldata proof
+    ) internal view {
+        ValidationData storage data = validationData[id];
+
+        /**
+         * @dev verify that sender is the same as in `newSignatureCommitment`
```

Comment:
Nit: You don't need to use the verbose `@dev` comment style within a function.

Reply:
https://github.com/Snowfork/polkadot-ethereum/pull/442/commits/dcc5b1468ca5e3618b1c07254dbb501b44aae014